v2.0/detailed-sql-support.md

```diff
@@ -0,0 +1,692 @@
+---
+title: Detailed SQL Standard Comparison
+summary: Details of CockroachDB's conformance to the SQL standard and which common extensions it supports.
+---
+
+<div id="toc"></div>
+
+## Overview
+
+This page lists which SQL standard features are supported, partially-supported, and unsupported by CockroachDB.
```

Comment:
I'd suggest simplifying this page by removing the toc and **Overview** headingh and starting the page with this:

> This page outlines to what extent CockroachDB conforms to the current SQL standard. 
> 
> - **Feature ID** ...
> - **Description** ...
> - **CockroachDB Support** ...

Reply:
Applies also the 1.1 version of the page.